RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Releasing
+
+Before releasing a new version of packages to the NPM and GitHub, Its advisable to follow the below steps in order
+
+- Update the dependencies `npm run clean`
+- Bootstrap the deps using `npm run bootstrap`
+- Make sure all tests passes `npm run test`
+
+### Releasing all packages
+
+To publish all the packages run `npm run publish`. It will run lerna publish (we use --independent) so it will prompt the version number for
+every package. `lerna publish` will take care of the following
```

Comment:
Yes lerna publish accepts

`[major | minor | patch | premajor | preminor | prepatch | prerelease]`

Reply:
lerna publish under the hood automatically determining a semantic version bump since we follow the conventional commits, 

But if we want to override this behaviour, We could still pass in like `npm run release -- (minor|major|patch)`

Reply:
Would you please add that to the RELEASE as well?